dbms/src/Server/Server.cpp

```diff
@@ -265,7 +287,12 @@ struct TiFlashProxyConfig
                 args_map[engine_store_address] = config.getString("flash.service_addr");
             else
                 args_map[engine_store_advertise_address] = args_map[engine_store_address];
-            args_map[engine_label] = engine_label_value;
+
+            auto disaggregated_mode = getDisaggregatedMode(config);
+            if (disaggregated_mode == DisaggregatedMode::Compute)
+                args_map[engine_label] = "tiflash_compute";
```

Comment:
When the engine_label is set to "tiflash_compute", is there any difference for the running process of TiFlashProxy?

Reply:
TiFlashProxy will report its label('tiflash_compute`) to PD, PD will not schedule region data to tiflash_compute node.